go.mod

```diff
@@ -1,4 +1,4 @@
-module github.com/m1/go-localize
+module github.com/cyx/go-localize
```

Comment:
Why was this changed?

Reply:
Just needed something for our project so we can move forward.